### PR TITLE
Enable persistent logs by default

### PR DESCRIPTION
**Pull request by Zempashi**
Fixes: #303

### CODE REVIEW

**Review by ballot-scality — DISMISSED** [review]
LGTM

File: roles/prepare_os/tasks/persistent_logs.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  stat:
+    path: '{{ persistent_logs_dir }}'
+  register: persistent_logs_dir_test
+
+- name: 'create {{ persistent_logs_dir }} if required'
+  file:
+    path: '{{ persistent_logs_dir }}'
+    state: directory
+  when:
+    - "persistent_logs_dir_test.stat.isdir == False"
```
Comment (NicolasT) [inline roles/prepare_os/tasks/persistent_logs.yml]: Consider
```
- not persistent_logs_dir_test.stat.isdir
- persistent_logs|bool
```

File: roles/prepare_os/tasks/persistent_logs.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - "persistent_logs|bool == True"
+
+- name: 'fail if {{ persistent_logs_dir }} exists and persistent_logs disabled'
+  fail:
+    msg: >-
+      By default journald enables persistent logs if {{ persistent_logs_dir }}
+      is present. We see that {{ persistent_logs_dir }} is present, but that
+      you have disabled persistent_logs. As we are cautious with your logs,
+      MetalK8s will not act to fix this. Please move or remove directory
```
Comment (NicolasT) [inline roles/prepare_os/tasks/persistent_logs.yml]: `act to fix` => `attempt to fix`
`remove directory` => `remove the directory`

File: roles/prepare_os/tasks/persistent_logs.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      yourself.
```
Comment (NicolasT) [inline roles/prepare_os/tasks/persistent_logs.yml]: `manually`

File: roles/prepare_os/tasks/persistent_logs.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  when:
+   - "persistent_logs_dir_test.stat.isdir == True"
```
Comment (NicolasT) [inline roles/prepare_os/tasks/persistent_logs.yml]: Consider
```
- persistent_logs_dir_test.stat.isdir
- not (persistent_logs|bool)
```

**Review by NicolasT — CHANGES_REQUESTED** [review]

**Comment by Zempashi** [comment]
@bert-e reset

**Review by ballot-scality — APPROVED** [review]

**Pull request merged**